client/Cargo.toml

```diff
@@ -91,7 +91,7 @@ irohad = { workspace = true }
 iroha_wasm_builder = { workspace = true }
 iroha_genesis = { workspace = true }
 test_network = { workspace = true }
-executor_custom_data_model = { path = "tests/integration/smartcontracts/executor_custom_data_model" }
+executor_custom_data_model = { version = "=2.0.0-pre-rc.21", path = "../wasm_samples/executor_custom_data_model" }
```

Comment:
Doesn't work without it, possibly because of [this](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#local-paths-in-published-crates). I found that both specifying the version and unnesting are necessary